TemplePlus/d20.cpp

```diff
@@ -1171,6 +1173,11 @@ ActionErrorCode D20ActionCallbacks::PerformUseItem(D20Actn* d20a){
 	return PerformCastSpell(d20a);
 }
 
+ActionErrorCode D20ActionCallbacks::PerformBreakFree(D20Actn* d20a) {
+	d20Sys.d20SendSignal(d20a->d20APerformer, D20DispatcherKey::DK_SIG_BreakFree, d20a->data1, 0);
```

Comment:
this

Reply:
I did not ignored what you said. I actually modified my initial code.

This button will be added by Web or Entangle. It will not appear otherwise. See Condition_sp_WebOn
When clicked,  it will send dispTypeD20Signal, Disp_Key_S_BreakFree. 

Any of these spells rely on the data1. It is simply not used.

Reply:
I'm using it for netted condition, thrown by Spiders. If you insist, I will undo the change, and use new Break free button, not this default action.

Reply:
Sorry for merging that guys.  I thought all the issues had been worked out and I didn't see anything else.  I'll make sure this gets fixed before releasing a new version.